debian/control.in

```diff
@@ -0,0 +1,27 @@
+Source: pgautofailover
+Section: database
+Priority: optional
+Maintainer: Citus Data <packaging@citusdata.com>
+Uploaders: Nils Dijk <nils@citusdata.com>
+Build-Depends: debhelper (>= 9), autotools-dev, flex, postgresql-server-dev-all (>= 158~), libedit-dev, libpam0g-dev, libselinux1-dev, libxslt1-dev
+Standards-Version: 3.9.7
+Homepage: https://github.com/citusdata/pg_auto_failover
+
+Package: pgautofailover-cli
```

Comment:
package name that will be used for `pg_autoctl`, let me know if you prefer an other name

Reply:
I know its a bit lengthy, but can we do `pg-auto-failover-cli` here?